src/endpoint.cpp

```diff
@@ -85,6 +86,58 @@ namespace oxen::quic
         _manual_routing = std::move(mrouting);
     }
 
+    void Endpoint::handle_ep_opt(opt::enable_0rtt_ticketing rtt)
+    {
+        _0rtt_enabled = true;
+        _0rtt_window = rtt.window.count();
+
+        _validate_0rtt_ticket = rtt._check ? std::move(rtt._check) : [this](gtls_ticket_ptr ticket, time_t current) -> bool {
+            auto key = ticket->key();
+
+            if (auto it = session_tickets.find(key); it != session_tickets.end())
+            {
+                if (auto exp = gnutls_db_check_entry_expire_time(it->second->datum()); current < exp)
+                {
+                    log::debug(log_cat, "Found existing anti-replay ticket for incoming connection; rejecting...");
+                    return GNUTLS_E_DB_ENTRY_EXISTS;
+                }
+
+                log::debug(log_cat, "Found expired anti-replay ticket for incoming connection");
+            }
+
+            session_tickets[std::move(key)] = std::move(ticket);
```

Comment:
```suggestion
            session_tickets[key] = std::move(ticket);
```
`key` is a string_view here, so moving it doesn't do anything special.